### PR TITLE
Improve contributing and add Bors all version test

### LINKED ISSUE #39: Bors.toml is missing integrations tests
Integrations tests should be  tested and successful en each bors merge

Tests should be made against each version of node

### PR DESCRIPTION
**Pull request by alallema**
closed #39

### CODE REVIEW

File: bors.toml [hunk 1]
```diff
@@ -1,3 +1,8 @@
-status = ['linter']
+status = [
+  'linter'
```
Comment (curquiza) [inline bors.toml:2]:
```suggestion
  'linter',
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -69,6 +71,11 @@ Launch Meilisearch instance:
 curl -L https://install.meilisearch.com | sh # download Meilisearch
 ./meilisearch --master-key=masterKey --no-analytics # run Meilisearch
 ```
+You may have need to [install Java](https://www.java.com/en/download/help/download_options.html) to run the emulator.</br>
```
Comment (curquiza) [inline CONTRIBUTING.md:74]:
```suggestion
You may need to [install Java](https://www.java.com/en/download/help/download_options.html) to run the emulator.</br>
```

**Review by bidoubiwa — APPROVED** [review]
LGTM 🔥🔥🔥

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**